packages/main/src/Menu.ts

```diff
@@ -400,15 +406,10 @@ class Menu extends UI5Element {
 		}
 		if (!this._isSubMenu) {
 			this._parentMenuItem = undefined;
+			this._opener = undefined;
 		}
 		const popover = await this._createPopover();
-		popover.initialFocus = "";
-		for (let index = 0; index < this._currentItems.length; index++) {
-			if (!this._currentItems[index].item.disabled) {
-				popover.initialFocus = `${this._id}-menu-item-${index}`;
-				break;
-			}
-		}
+		popover.initialFocus = `${this._id}-menu-item-0`;
```

Comment:
Will it work if there are no menu items?

Reply:
Yes, that is a property of the ui5-popup. The required check is there.